screens/Home/MyVcs/GetIdInputModal.tsx

```diff
@@ -67,6 +71,7 @@ export const GetIdInputModal: React.FC<GetIdInputModalProps> = props => {
                     <Centered width={32} fill>
                       {controller.isInvalid ? (
                         <Icon
+                          {...testIDProps('getIdError')}
```

Comment:
Let's try to add testID in popover element of Tooltip `popover={<Text>{t('qstnMarkToolTip')}</Text>}`